addons/editor-dark-mode/addon.json

```diff
@@ -962,6 +962,34 @@
         "darkScrollbars": true
       }
     },
+    {
+      "name": "TurboWarp Dark",
+      "id": "tw-dark",
+      "description": "A dark theme based on TurboWarp's dark mode.",
+      "values": {
+        "page": "#111111",
+        "primary": "#4d97ff",
+        "highlightText": "#4d97ff",
```

Comment:
I think using the TurboWarp red is appropriate for a preset with TurboWarp in its name.

Reply:
OK, I was originally going to make it red, but it changed somethings that TurboWarp kept blue like selecting objects things in the paint editor and somethings are red in TuboWarp, but can't be changes with this addon, like the buttons on the top of the paint editor.

Reply:
Changing the color of icons is something I want to implement in the future. TurboWarp not changing the outline of selected objects seems more like a missing feature to me (it does change the selection style in the sound editor).